### PR TITLE
Beta prep

### LINKED ISSUE #669: Updates and impact on metrics
Good point raised by @microbit-carlos below. Discussion required on the best direction to take.

> Once we release V3 as beta, how often would we expect to deploy with updates?
The metrics currently create a new category based on the Editor version, so right now in alpha each category only contains a few days of data.
In a slow release schedule, like what we had with V2, that has worked fine, but maybe we want to change it in V3? As an option, we could only create a new category when we make changes to the metrics (or new UI widgets are added, which might add new metrics)?

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
- Changes the alpha release notice to a beta one.
- Bumps the theme pages to a version with the simple fix that closes #669.

### CODE REVIEW

**Comment by microbit-matt-hillsdon** [comment]
Link directly to the "Guide" page and label it "Guidance". The Help & support link remains the same.
MicroPython Documentation -> MicroPython documentation for consistency.

File: src/workbench/InfoDialog.tsx [hunk 1]
```diff
@@ -36,58 +47,64 @@ const InfoDialog = ({
...
                   <ListItem m={1}>
-                    Autocomplete and signature help in the code editor.
+                    Working code examples in the <strong>Reference</strong> tab.
+                    Drag and drop them into your code.
                   </ListItem>
                   <ListItem m={1}>
-                    Drag and drop code examples from the{" "}
-                    <strong>Reference</strong> and <strong>API</strong>{" "}
-                    documentation tabs.
+                    Autocomplete and paramerer help. See the choices available
```
Comment (microbit-robert) [inline src/workbench/InfoDialog.tsx:64]:
```suggestion
                    Autocomplete and parameter help. See the choices available
```

**Review by microbit-robert — CHANGES_REQUESTED** [review]
See suggestion for typo fix. Otherwise this looks good.

**Comment by microbit-matt-hillsdon** [comment]
> See suggestion for typo fix. Otherwise this looks good.

Good catch, thanks @microbit-robert.

**Pull request merged**